cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillRunnerService.java

```diff
@@ -153,8 +152,6 @@ public class KubeTwillRunnerService implements TwillRunnerService, NamespaceList
   private final Map<String, Map<Type, AppResourceWatcherThread<?>>> resourceWatchers;
   private final Map<String, KubeLiveInfo> liveInfos;
   private final Lock liveInfoLock;
-  private final int jobCleanupIntervalMins;
-  private final int jobCleanBatchSize;
   private final String selector;
   private final boolean enableMonitor;
   private final Gson gson;
```

Comment:
I notice that this field is unused. Do we need it?

Reply:
removed this unused field